react-design/my-app/src/ButtonClick.js

```diff
@@ -0,0 +1 @@
+export default function ButtonClick() {}
```

Comment:
To avoid confusion, try not to capitalize your functions unless they're meant to be components